app/views/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1> Showing posts for user with id:<%= @user_id %></h1>
```

Comment:
- [ ] As required for the project, kindly make sure you have corresponding view files for each of the controller actions. Right now, I noticed that you miss the `Post#show` (i.e. `# app/views/posts/show.html.erb`) view page. I suggest you add it to match the project's requirements.